### PR TITLE
feat: support 2.1.0 spec version

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Support 2.1.0 spec version:
- render the new shape of message's examples with `name` and `summary`.

Example using the gemini example:

```md
> Examples of payload

**updateMessage**

Example of an update message that contains a change in price information.

```json
{
  "type": "update",
  "eventId": 36902233362,
  "timestamp": 1619769673,
  "timestampms": 1619769673527,
  "socket_sequence": 661,
  "events": [
    {
      "type": "change",
      "side": "bid",
      "price": "54350.40",
      "remaining": "0.002",
      "delta": "0.002",
      "reason": "place"
    }
  ]
}
``


**heartbeatMessage**

Example of additional heartbeat message when you enable them.

```json
{
  "type": "heartbeat",
  "socket_sequence": 1656
}
``

```

**Related issue(s)**
Part of https://github.com/asyncapi/spec/issues/536

### CODE REVIEW

File: components/Message.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { Text } from "@asyncapi/generator-react-sdk";
-import { generateExample, getHeadersExamples, getPayloadExamples } from "@asyncapi/generator-filters";
```
Comment (derberg) [inline components/Message.js:2]: can you explain this change to a custom functions

Comment (magicmatatjahu) [reply components/Message.js:2]: `getHeadersExamples` or `getPayloadExamples` only return the `examples[*].headers` or `examples[*].payload`, so we don't have corresponding `name` or `summary` for appropriate example. If we will have change from PR in `@asyncapi/generator-filters` we must know that then it's a breaking change and we must update check all templates where we use the `getHeadersExamples` and `getPayloadExamples`. Making changes only here is safe.

Comment (derberg) [reply components/Message.js:2]: `getHeadersExamples` or `getPayloadExamples` are more than what you wrote, if examples are not in `examples` object, they are taken from payload or headers schema (if provided of course). 

we can release generator-filters 2.0 🤷🏼 unless you already have your own versions of these functions in react component. Then we are just looking an effort we did in the past to move generic functions to `@asyncapi/generator-filters`

Comment (magicmatatjahu) [reply components/Message.js:2]: > getHeadersExamples or getPayloadExamples are more than what you wrote, if examples are not in examples object, they are taken from payload or headers schema (if provided of course).

Functions in PR also are looking for that :)

> we can release generator-filters 2.0 🤷🏼 unless you already have your own versions of these functions in react component. Then we are just looking an effort we did in the past to move generic functions to @asyncapi/generator-filters

So I create PR in `generator-filters`.

Comment (magicmatatjahu) [reply components/Message.js:2]: @derberg Created https://github.com/asyncapi/generator-filters/pull/41

**Review by derberg — APPROVED** [review]

**Review by derberg — APPROVED** [review]

**Pull request merged**